docs/extensibility.md

```diff
@@ -449,17 +482,47 @@ spec:
 ### Installing Akri with your new Configuration
 Before you can install Akri and apply your Nessie Configuration, you must first build both the Controller and Agent containers and push them to your own container repository. You can use any container registry to host your container repository.We are using the new [GitHub container registry](https://github.blog/2020-09-01-introducing-github-container-registry/). If you want to enable GHCR, you can follow the [getting started guide](https://docs.github.com/en/free-pro-team@latest/packages/getting-started-with-github-container-registry). 

```

Comment:
This seems redundant to the precious step which talks about creating a GHCR for your broker. Maybe reference that same registry here or when discussing where to store the broker also mention that you can later store the Agent and Controller there.

Reply:
👍